libs/ui/src/table.tsx

```diff
@@ -27,10 +27,10 @@ export const Table = styled.table<TableProps>`
   }
   & th {
     border-top: 1px solid ${tableBorderColor};
-    font-size: 0.75rem;
```

Comment:
I like this change! Have you spot checked other screen/modals with tables to make sure it looks good in other contexts as well? I know we use them in a lot of places so it might be worth a glance

Reply:
Yeah, good nudge! I'd spot-checked a few screens in VxAdmin, but will also take a look at one or two other apps, just to be safe.

Also just noticed I should've placed the print media override within this `th` block to properly preserve previous behaviour for prints, so I'll fix that up